Reference/Notifications/CacheRefresher-Notifications/index.md

```diff
@@ -0,0 +1,178 @@
+---
+versionFrom: 9.0.0
+versionTo: 11.0.0
+---
+
+# Cache Refresher Notifications
+
+Before starting with cache refresher notifications it's a good idea to ensure you need to use these, if you want to react to changes in content for instance, there's no real reason to use these notifications, since the [content service notifications](../ContentService-Notifications/index.md) will be easier to work with, but if you need to react to changes in the cache, then these are the notifications for you.
+
+Cache refresher notifications are sent when the cache has refreshed, there's multiple different type of cache refresher notifications, based on what type has been updated in the cache, for instance content or media. All these notifications inherit from the same base notification: `CacheRefresherNotification`, the base notification is implemented in the following way:
+
+```C#
+public abstract class CacheRefresherNotification : INotification
+{
+    public CacheRefresherNotification(object messageObject, MessageType messageType)
+    {
+        MessageObject = messageObject ?? throw new ArgumentNullException(nameof(messageObject));
+        MessageType = messageType;
+    }
+
+    public object MessageObject { get; }
+
+    public MessageType MessageType { get; }
+}
+```
+
+As you can see this notification contains two properties, a `MessageObject` and a `MessageType`. The `MessageType` specifies what kind of cache operation was performed, for instance `RemoveById`, the possible message types is as follows: 
+
+```C#
+public enum MessageType
+{
+    RefreshAll,
+    RefreshById,
+    RefreshByJson,
+    RemoveById,
+    RefreshByInstance,
+    RemoveByInstance,
+    RefreshByPayload,
+}
+```
+The other parameter `MessageObject` will depend on what type of cache refresher notification you're handling, if you for instance handle the `ContentCacheNotification` the message object will be `ContentCacheRefresher.JsonPayload[]`
+
+This object contains the Id and key of the item being updated, as well as an enum specifying how the tree is updated:
```

Comment:
```suggestion
This object contains the Id and key of the item being updated, as well as an enumerable specifying how the tree is updated:
```

Reply:
Here I do mean the type `enum` and not an enumerable, this can also be written as enumeration, but it's usually just called enum